app/src/main/java/pm/gnosis/heimdall/ui/onboarding/SetupSafeIntroActivity.kt

```diff
@@ -0,0 +1,49 @@
+package pm.gnosis.heimdall.ui.onboarding
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import com.jakewharton.rxbinding2.view.clicks
+import io.reactivex.rxkotlin.plusAssign
+import io.reactivex.rxkotlin.subscribeBy
+import kotlinx.android.synthetic.main.layout_finish_setup.*
+import pm.gnosis.heimdall.HeimdallApplication
+import pm.gnosis.heimdall.R
+import pm.gnosis.heimdall.common.di.components.DaggerViewComponent
+import pm.gnosis.heimdall.common.di.modules.ViewModule
+import pm.gnosis.heimdall.common.utils.startActivity
+import pm.gnosis.heimdall.ui.base.BaseActivity
+import pm.gnosis.heimdall.ui.safe.add.AddSafeActivity
+import pm.gnosis.heimdall.ui.safe.overview.SafesOverviewActivity
+
+class SetupSafeIntroActivity : BaseActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        inject()
+        setContentView(R.layout.layout_finish_setup)
+    }
+
+    override fun onStart() {
+        super.onStart()
```

Comment:
If you press back in this screen you will not see it again, right? Didn't we want to force interaction with either button?

Reply:
I think it's unnecessary to have one more check for this in the Splash screen. If the users exit the app at this point when they return they will have the safe overview with the action to add a new safe.